### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by GauthierPLM**

### CODE REVIEW

File: docs/getting-started-using-a-plan.md [hunk 1]
```diff
@@ -2,108 +2,145 @@
...
+*Note: for the sake of this tutorial, we are browsing
+[the Molecule repository](https://github.com/alanzanattadev/atom-molecule-dev-environment/)*
 
-The main addition is the **Control Panel**, which is located at the bottom of
-the screen by default. On the left of this panel is the **Plugin Subpanel**.
+The main addition of Molecule is the **Control Panel**, which is located at the bottom of
+the screen by default. It allows you to control the execution of tools on your project through plugins,
+and offers you an easy-to-read, fancy view of their output.
 
-Plugin Subpanel
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md:14]: "Panel" has a specific meaning in Atom (the widgets on the sides of the screen), which the "list of plugins" widget doesn't match, hence why it should have another name.

File: docs/getting-started-using-a-plan.md [hunk 2]
```diff
@@ -2,108 +2,145 @@
...
+
+Each plugin is displayed with a list of  pre-generated plans and an option to
+[create a new plan](getting-started-creating-a-plan.md).
 
-Each plugin is displayed with a list of plugin-specific, pre-generated plans,
-and an option to [create a new plan](creating-a-plan.md).
+Each plan is displayed with three buttons:
 
-Each plan is displayed with two buttons:
+- A **Split** button that will split the Control Panel in two part and will open the
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: *two parts

*This allows you

"and easily move the Control Panel wherever you want in Atom" How so? You can move the Control Panel without splitting it.

File: docs/getting-started-using-a-plan.md [hunk 3]
```diff
@@ -2,108 +2,145 @@
...
 
-- The Diagnostics button opens the [Diagnostics Subpanel](#diagnostics-subpanel).
+- The Diagnostics button opens the [Diagnostics Panel](#diagnostics-panel).
+
+- The Terminal button opens the [Terminal Panel](#terminal-panel).
 
-- The Terminal button opens the [Terminal Subpanel](#terminal-subpanel).
+#### Watch Mode
 
-As you may have noticed, the Jest plan "Watch mode" runs in, well, watch mode:
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: Why the change? I think the old line is easier to understand for someone who doesn't know Molecule.

*you did not break anything

File: docs/getting-started-using-a-plan.md [hunk 4]
```diff
@@ -2,108 +2,145 @@
...
-Diagnostics Subpanel
+Diagnostics Panel
 --------------------
 
 All Molecule plugins have the same workflow:
 
-* They analyse your project's files.
-* They execute some operation on these files
-* They output a stream of plugin-dependent messages.
+* They analyse your project's files to find the files they are interested by ;
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: *the files they are interested in

I'd suggest "(syntax analysis, unit tests, etc)"

File: docs/getting-started-using-a-plan.md [hunk 5]
```diff
@@ -2,108 +2,145 @@
...
+* They output a stream of messages to describe their result ;
 
 These messages are called **Diagnostics**, and can be accessed in the
-**Diagnostics Subpanel**.
+**Diagnostics Panel**.
 
-![The Diagnostics Subpanel](assets/diagnostics-closeup.png)
+![The Diagnostics Panel](assets/diagnostics-closeup.png)
 
-Once you've executed a plan, the Diagnostics Subpanel will start to
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: *will start to fill with Diagnostics

*There are several types of these

(I'd say you should keep the previous wording)

File: docs/getting-started-using-a-plan.md [hunk 6]
```diff
@@ -2,108 +2,145 @@
...
 
-Clicking on a diagnostic will lead you to the place in the code it concerns. For
+Clicking on a diagnostic will jump to the code it concerns. For
 instance, clicking on a syntax error diagnostic will open the relevant file, and
-place your cursor at the beginning of the error.
+place your cursor at the line of the error.
+
+### Filtering diagnostics
+
+Molecule allows you to easily filter the diagnostics of your project.
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: Words and phrases like "easily", "simply", "you just have" are bloat.

You don't need to tell the user that clicking a button is really easy. Just say "You can filter the diagnostics", "Click the icon to clear the search", "click again", without any adverbs, unless the adverbs convey additional meaning.

File: docs/getting-started-using-a-plan.md [hunk 7]
```diff
@@ -2,108 +2,145 @@
...
+You can either hide by severity by clicking on the corresponding icons, next to which
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: When writing a manual, avoid this kind of sentence structure: "You can do X by using Y, which is a Z and a W". It'll confuse readers, who'll have to read the sentence several times to get what you mean.

Instead say "Y is a Z and a W. You can do X by using Y".

Here, the appropriate structure would be

`There are several icons displayed, that represent the different levels of diagnostic severity. For each severity level, the number of diagnostics with that severity is displayed next to the matching icon. You can click on each icon to toggle showing diagnostics with the matching severity level.`

`You can also search for a specific term. Only diagnostics with content matching the search term will displayed.`

Also, adding specific examples would help. ("For instance, if you type "react" in the search bar, only diagnostics with "react" in their message will be displayed")

File: docs/getting-started-using-a-plan.md [hunk 8]
```diff
@@ -2,108 +2,145 @@
...
-concerned plan.
+While plugins transform the text these commands output into a more readable
+format, sometimes you may want to read the raw output directly. You achieve this by
+by clicking on the **Terminal** button to switch to the **Terminal Panel**.
 
 ![The Terminal Panel](assets/terminal-closeup.png)
 
-Some plugins may ask you to enter text into the Terminal Panel (ex: a password).
-However, in most cases, interacting with the Terminal is impossible.
+Some tools may need user interaction to work (by entering some text or password for
```
Comment (PoignardAzur) [inline docs/getting-started-using-a-plan.md]: Superfluous "easily".

File: docs/getting-started-using-a-plan.md (hunk 3 above)
Comment (GauthierPLM) [reply docs/getting-started-using-a-plan.md]: Because it's focused on Jest and may lead the person to think that it's the only tool supporting it, which is not the case.
So I chose to present the principle and then use an exemple.

File: docs/getting-started-using-a-plan.md (hunk 7 above)
Comment (GauthierPLM) [reply docs/getting-started-using-a-plan.md]: Thank you, your wording is indeed much better.
I'll add an image to properly illustrate this part of the documentation as it might be abstract.
